library/src/commonMain/kotlin/org/jraf/klibnotion/internal/api/model/block/ApiBlockBookmark.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * This source is part of the
+ *      _____  ___   ____
+ *  __ / / _ \/ _ | / __/___  _______ _
+ * / // / , _/ __ |/ _/_/ _ \/ __/ _ `/
+ * \___/_/|_/_/ |_/_/ (_)___/_/  \_, /
+ *                              /___/
+ * repository.
+ *
+ * Copyright (C) 2021-present Benoit 'BoD' Lubek (BoD@JRAF.org)
```

Comment:
Don't hesitate to add your own copyright notice on top of this line for files you touched, and instead of mine in files you created!

Reply:
I figured out some Intellij tricks and batch inserted a line in all touched files.